### PR TITLE
Added new feature: customize day header in month view.

### PR DESCRIPTION
**Pull request by Daavidaviid**
Can now add custom components day header in month view, and each custom components will receive the date of the day in the props.
capture d ecran 2017-12-15 a 14 25 09

It will be useful for my bros and I. The react game is merciless.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-big-calendar",
-  "version": "0.17.0",
+  "version": "0.17.1",
```
Comment (jquense) [inline package.json]: please don't bump the version, that gets handled when the package is published

**Comment by jquense** [comment]
you got a lot of noise in the PR. please cut it down to just the code changes for the feature, thanks!

**Comment by Daavidaviid** [comment]
@jquense Ok, sorry I was experimenting with it. I removed all the useless stuff.

**Comment by jquense** [comment]
Sorry I think this already exists, or at least can be handled by something that already exists. `components.dateHeader` is implemented already for this purpose

**Pull request closed without merging**